From: Andrew Morton <akpm@linux-foundation.org>
Date: Fri, 29 Nov 2024 21:23:51 -0800
Subject: [PATCH] 
 mm-page_alloc-add-some-detailed-comments-in-can_steal_fallback-fix

tweak grammer, fit to 80 cols

Cc: gaoxiang17 <gaoxiang17@xiaomi.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/page_alloc.c | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/mm/page_alloc.c b/mm/page_alloc.c
index eca826045410..cc3296cf8c95 100644
--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -1854,11 +1854,12 @@ static bool can_steal_fallback(unsigned int order, int start_mt)
 		return true;
 
 	/*
-	 * movable pages won't cause permanent fragmentation, so when you alloc small pages,
-	 * you just need to temporarily steal unmovable or reclaimable pages that are closest
-	 * to the request size. After a while, memory compact may occur to form large contiguous
-	 * pages, and the next movable allocation may not need to steal. Unmovable and reclaimable
-	 * allocation need to actually steal pages.
+	 * Movable pages won't cause permanent fragmentation, so when you alloc
+	 * small pages, you just need to temporarily steal unmovable or
+	 * reclaimable pages that are closest to the request size.  After a
+	 * while, memory compaction may occur to form large contiguous pages,
+	 * and the next movable allocation may not need to steal.  Unmovable and
+	 * reclaimable allocations need to actually steal pages.
 	 */
 	if (order >= pageblock_order / 2 ||
 		start_mt == MIGRATE_RECLAIMABLE ||
-- 
2.39.5

